dist/licenses/license-cloud.google.com-go-bigquery.txt

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
Do not blindly copy all the files generated by license-eye, for standard APL2.0 license, we don't need to copy their license files

Reply:
Deleted, should we need to add a parameter to ignore the licenses when saving the license files?

Reply:
> Deleted, should we need to add a parameter to ignore the licenses when saving the license files?

This is not a general use case, and only some ASF projects do this, we can add a filter to only save licenses satisfying the filter